apps/website/.vuepress/theme/components/Home.vue

```diff
@@ -107,15 +107,25 @@
       <h3 cds-text="display center medium" cds-layout="m-b:xl">Popular Resources</h3>
       <div cds-layout="grid gap:lg cols@xs:12 cols@sm:6">
         <div class="image-cell" cds-layout="m-b:sm m-b@sm:xxl">
-          <img cds-layout="container:fill" src="/images/home/img-pr-components.svg" aria-hidden="true" />
-          <h4 cds-text="title medium" cds-layout="m-t:lg">Components</h4>
+          <a href="/components/accordion/">
+            <img cds-layout="container:fill" src="/images/home/img-pr-components.svg" aria-hidden="true" />
```

Comment:
If we hide the images with aria-hidden, we need some offscreen text here. Can't we just have alt text instead?

Reply:
I haven't even noticed `aria-hidden` attributes were there

Reply:
All home page images need to have these `aria-hidden` removed. But I avoided making those changes in this PR as it might cause conflicts with Matt's work on the home page.